pennylane/ftqc/pauli_tracker.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""
+This module contains Pauli Tracking functions.
+"""
+
+from typing import List, Tuple
+
+import numpy as np
+
+from pennylane import I, X, Y, Z
+from pennylane.operation import Operator
+
+_OPS_TO_XZ = {
+    I: (0, 0),
+    X: (1, 0),
+    Y: (1, 1),
+    Z: (0, 1),
+}
+
+_XZ_TO_OPS = {
+    (0, 0): I,
+    (1, 0): X,
+    (1, 1): Y,
+    (0, 1): Z,
+}
+
+_PAULIS = frozenset({X, Y, Z, I})
+
+
+def pauli_to_xz(op: Operator) -> Tuple[np.uint8, np.uint8]:
+    r"""
+    Convert a `Pauli` operator to its `xz` representation up to a global phase, i.e., :math:`encode_{xz}(Pauli)=(x,z)=X^xZ^z)`, where
+    :math:`x` is the exponent of the :class:`~pennylane.X` and :math:`z` is the exponent of
+    the :class:`~pennylane.Z`, meaning :math:`encode_{xz}(I) = (0, 0)`, :math:`encode_{xz}(X) = (1, 0)`,
+    :math:`encode_{xz}(Y) = (1, 1)` and :math:`encode_{xz}(Z) = (0, 1)`.
+
+    Args:
+        op (qml.operation.Operator): A Pauli operator.
+
+    Return:
+        A tuple of xz encoding data, :math:`x` is the exponent of the :class:`~pennylane.X`, :math:`z` is the exponent of
+    the :class:`~pennylane.Z`.
```

Comment:
```suggestion
        A tuple of xz encoding data, :math:`x` is the exponent of the :class:`~pennylane.X`, :math:`z` is the exponent of
        the :class:`~pennylane.Z`.
```

Otherwise it gets put on a new line